api/src/pcapi/scripts/populate_collective_offer_formats/main.py

```diff
@@ -0,0 +1,21 @@
+from pcapi.app import app
+from pcapi.core.categories.models import EacFormat
+from pcapi.core.educational import models as educational_models
+from pcapi.models import db
+
+
+def populate() -> None:
+    collective_offers = educational_models.CollectiveOffer.query.filter(
+        educational_models.CollectiveOffer.formats == None
+    ).all()
+    for collective_offer in collective_offers:
+        collective_offer.formats = [EacFormat.CONCERT]
+        db.session.add(collective_offer)
```

Comment:
Je pense que tu pouvais tout faire en une seule requête : 
```suggestion
    collective_offers = educational_models.CollectiveOffer.query.filter(
        educational_models.CollectiveOffer.formats.is_(None)
    ).update(
        {"formats": [EacFormat.CONCERT]},
        synchronize_session=False,
    )
```
mais pour un script de rattrapage sur integration, ça n'a pas vraiment d'importance :slightly_smiling_face:

Reply:
Oui c'était ma première version mais bizarrement ça n'a pas marché. J'ai pas beaucoup creusé pourquoi mais ça a bien marché.